docs/root/configuration/http/http_filters/grpc_json_transcoder_filter.rst

```diff
@@ -104,3 +104,18 @@ gRPC or RESTful JSON requests to localhost:51051.
 
 .. literalinclude:: _include/grpc-transcoder-filter.yaml
     :language: yaml
+
+
+Modified Envoy configuration to use gRPC reflection
+---------------------------------------------------
+Rather than using a statically generated configuration, Envoy can fetch service
+definitions via grpc reflection. This can be done by replacing
+``proto_descriptor`` in the filter's config with the following:
```

Comment:
Does this imply that the `proto_descriptor` and `reflection_cluster_config` are mutually exclusive?
if so this should be clarified in the API (proto comments).

Reply:
Yes. I will make sure to make it clear in the comments.